annotations/tekton-annotations/src/main/java/io/dekorate/tekton/step/KanikoBuildStep.java

```diff
@@ -24,20 +24,15 @@
 public class KanikoBuildStep extends ImageBuildStep<KanikoBuildStep> {
 
   public static final String DOCKERFILE_FORMAT = "--dockerfile=%s";
-  public static final String DOCKERFILE_ARG = "--dockerfile=$(inputs.params." + PATH_TO_DOCKERFILE_PARAM_NAME + ")";
   public static final String CONTEXT_FORMAT = "--context=%s";
-  public static final String CONTEXT_ARG = "--context=$(params." + PATH_TO_CONTEXT_PARAM_NAME + ")";
   public static final String WORKSPACE_SOURCE = "/workspace/source";
   public static final String FORCE_ARG = "--force";
   public static final String INSECURE_ARG = "--insecure";
-  public static final String IMAGE_DESTINATION_ARG = "--destination=$(resources.outputs.image.url)";
```

Comment:
I see in many places replacing `$(resources.outputs.image.url)` it's not clear to me why we do this.
Is it because the `output` resource is no longer available ? Is it something else?

Reply:
The `$(resources...` is deprecated (and removed in Tekton v0.47 which is the version that the Fabric8 Kubernetes Client 6.7.2 uses). The replacement is to use the params.